# Thanks to czk <msieczka(at)sieczka.org> for providing the patch inside aur/grass64 PKGBUILD 
--- ./gui/wxpython/core/globalvar.py	2013-07-10 00:09:27.000000000 +0200
+++ ./gui/wxpython/core/globalvar.py.wxversion_hack	2014-01-25 15:45:02.781992175 +0100
@@ -51,6 +51,7 @@
         except ImportError, e:
             raise ImportError(e)
         # wxversion.select(str(minVersion[0]) + '.' + str(minVersion[1]))
+        wxversion.select('2.8')
         wxversion.ensureMinimal(str(minVersion[0]) + '.' + str(minVersion[1]))
         import wx
         version = wx.version().split(' ')[0]
